Libraries/VaahStripe.md

```diff
@@ -106,14 +194,103 @@ Reference url: https://stripe.com/docs/api
         'state'         => 'xxxxxx'                     // optional
     ];
 
-    $price_id      // Price define the unit cost, currency, and (optional) billing cycle for Subcription   
+    $price_id    You can generate Price ID from createPrice Method of Vaah Stripe                                 // Price define the unit cost, currency, and (optional) billing cycle for Subcription   
+    or You can create directly from Stripe Dashboard by visit https://dashboard.stripe.com/test/products/create 
     
-    $return_url    // URL to redirect your customer back to after they authenticate or cancel their payment
+    $return_url = http://localhost/vaahcms/public/      // URL to redirect your customer back to after they authenticate or cancel their payment
 
     \VaahStripe::subscription($customer, $card, $address, $price_id, $return_url);
 
 ```
 
+Success Response 
+
+```
+{
+    "status": "success",
+    "data": {
+        "id": "pi_1LMwxxxxxxxxxxxxxxxxxxxx",                // Payment Intent ID
+        "object": "payment_intent",
+        "allowed_source_types": [
+            "card"
+        ],
+        "amount": 79900,
+        "amount_capturable": 0,
+        "amount_details": {
+            "tip": []
+        },
+        "amount_received": 0,
+        "application": null,
+        "application_fee_amount": null,
+        "automatic_payment_methods": null,
+        "canceled_at": null,
+        "cancellation_reason": null,
+        "capture_method": "automatic",
+        "charges": {
+            "object": "list",
+            "data": [],
+            "has_more": false,
+            "total_count": 0,
+            "url": "/v1/charges?payment_intent=pi_1LMwxxxxxxxxxxxxxxxxxxxx"
+        },
+        "client_secret": "pi_1LMwxxxxxxxxxxxxxxxxxxxx_secret_usv8FrEvzU0CuWEyRjmUKlmjD",
+        "confirmation_method": "automatic",
+        "created": 1658161603,
+        "currency": "usd",
+        "customer": "cus_M570qAPnoC6xl1",
+        "description": "Subscription creation",
+        "invoice": "in_1LMwmxxxxxxxxxxxxxxxxxxx",
+        "last_payment_error": null,
+        "livemode": false,
+        "metadata": [],
+        "next_action": {
+            "redirect_to_url": {
+                "return_url": "http://localhost/vikram/vaahcms-dev-env/public",
+                "url": "https://hooks.stripe.com/3d_secure_2/hosted?merchant=                                               // Url to complete your 3D secure Payment or open OTP Page
```

Comment:
@wri-vikram-wri231 there will be 2 cases, 3D security required and not required, explain both cases and response. Display only the parameters for which explanation is required.